src/java.base/share/classes/java/util/Comparator.java

```diff
@@ -189,6 +188,42 @@ default Comparator<T> reversed() {
         return Collections.reverseOrder(this);
     }
 
+    /**
+     * Returns the greater of two values according to this comparator.
+     * If the arguments are equal with respect to this comparator,
+     * the {@code a} argument is returned.
+     *
+     * @param   a   an argument.
+     * @param   b   another argument.
+     * @return  the larger of {@code a} and {@code b} according to this comparator.
+     * @throws  ClassCastException if the collection contains elements that are
+     *          not <i>mutually comparable</i> (for example, strings and
```

Comment:
`<em>` is more intent revealing.
```suggestion
     *          not <em>mutually comparable</em> (for example, strings and
```

Reply:
I copied this sentence from `java.util.Collections` where `<i>` is used 48 times. I'll replace it here with `<em>`, but probably it should be updated en-masse for consistency?